projects/angular/src/forms/combobox/combobox.ts

```diff
@@ -257,14 +257,17 @@ export class ClrCombobox<T>
   }
 
   onBlur() {
-    this.onTouchedCallback();
-    if (this.control.control.updateOn === 'change' && this.control.control?.errors?.required) {
+    this.focused = false;
+
+    this.onTouchedCallback?.();
+
+    if (this.control?.control?.updateOn === 'change' && this.control?.control?.errors?.required) {
```

Comment:
```suggestion
    if (this.control?.control?.updateOn === 'change' && this.control.control.errors?.required) {
```

This has unnecessary conditional access operators. It is *impossible* for these properties to be `undefined` or `null` in the second condition.

I'm going to take a closer look at this because I think there are unnecessary conditional access operators added in this PR that are not needed. The current code doesn't have these (and the one on this line in current is impossible to hit).

Is this an issue in the current code? Or what in this change introduces the need for conditional access? I think I'm missing something.

Reply:
I misunderstood your statement.

Corrected in commit aef0752